.github/workflows/autograding.yml

```diff
@@ -0,0 +1,71 @@
+# This is a basic workflow to help you get started with Actions
+
+name: Autograding
+
+# Controls when the workflow will run
+on:
+  # Triggers the workflow on push or pull request events but only for the "main" branch
+  push:
+    branches: [ "main", "master" ]
```

Comment:
Please check lab1 changes. 
I removed these to reduce the number of builds. 
Only pull_request is sufficient since Feedback PR is created by default.